DigitalLearningSolutions.Web/Models/Enums/FallbackDataUsedWarningText.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace DigitalLearningSolutions.Web.Models.Enums
+{
+    using DigitalLearningSolutions.Data.Enums;
+
+    public class FallbackDataUsedWarningText : Enumeration

```

Comment:
Nice that you've pulled these messages out. Is there a particular reason you've made it an enumeration? It feels like it's just a static helper job to me. When I see an enumeration I generally expect it to be a list of possible values of something (e.g. subapplications of DLS or types of user or states of a video player - paused, FF, playing etc.). Enumeration can definitely work here, but it feels a bit heavyweight.

Reply:
Oh nice, I didn't think of a static helper. I've moved these warning texts to a static helper class, along with the other Learning hub-related warning texts I could find